From 2a24432d3fa7f359dc2a9b7e803c657ee434ecc3 Mon Sep 17 00:00:00 2001
From: Ankit Agarwal <ankit.agarwal@intel.com>
Date: Tue, 18 Jun 2024 10:44:39 +0530
Subject: [PATCH] [WA] Fixed Sepolicy issue.

Observed build issue due to never allow sepolicy for type attribute
system_device_block.
Removing for now.

Tests:  Build EB, there was no error.

Tracked-On: NA
Signed-off-by: Ankit Agarwal <ankit.agarwal@intel.com>
---
 boot-arch/generic/device.te     | 2 +-
 boot-arch/generic/file_contexts | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/boot-arch/generic/device.te b/boot-arch/generic/device.te
index a35611f..c4c9736 100644
--- a/boot-arch/generic/device.te
+++ b/boot-arch/generic/device.te
@@ -3,6 +3,6 @@ type acpio_block_device, dev_type;
 type vendor_block_device, dev_type;
 type product_block_device, dev_type;
 type odm_block_device, dev_type;
-typeattribute system_block_device super_block_device_type;
+#typeattribute system_block_device super_block_device_type;
 typeattribute vendor_block_device super_block_device_type;
 typeattribute product_block_device super_block_device_type;
diff --git a/boot-arch/generic/file_contexts b/boot-arch/generic/file_contexts
index 4984f6c..af91863 100644
--- a/boot-arch/generic/file_contexts
+++ b/boot-arch/generic/file_contexts
@@ -4,7 +4,7 @@
 /dev/block/by-name/boot(_(a|b))?	u:object_r:boot_block_device:s0
 /dev/block/by-name/bootloader(_(a|b))?	u:object_r:boot_block_device:s0
 /dev/block/by-name/multiboot(_(a|b))?	u:object_r:boot_block_device:s0
-/dev/block/by-name/system(_(a|b))?	u:object_r:system_block_device:s0
+#/dev/block/by-name/system(_(a|b))?	u:object_r:system_block_device:s0
 /dev/block/by-name/vendor(_(a|b))?	u:object_r:vendor_block_device:s0
 /dev/block/by-name/product(_(a|b))?	u:object_r:product_block_device:s0
 /dev/block/by-name/odm(_(a|b))?	u:object_r:odm_block_device:s0
-- 
2.34.1

